### ISSUE TITLE
9.1.3.1a/11.1 On the page, there are two "h1" headings, where only the "Documents" heading makes sense for structuring content. It is recommended to remove the "All files" heading.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42275]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-be75bbae42-v11-n1

9.1.3.1a/11.1 On the page, there are two "h1" headings, where only the "Documents" heading makes sense for structuring content. It is recommended to remove the "All files" heading.

**Comment by JuliaKirschenheuter** [comment]
Dear @michaelnissenbaum,

i would like to clarify this issue with you. Some time ago we have structured our headings based on the section in a left sidebar (navigation), please have a look https://github.com/nextcloud/server/issues/36914. This logic is remaining through all our apps and pages.

![Screenshot from 2024-01-12 11-02-49](https://github.com/nextcloud/server/assets/6078378/c5f793db-e7da-4319-bcee-5be2d05cd1e8)

Should we follow another approach?

Thank you!

**Comment by michaelnissenbaum** [comment]
In this case, you can follow the same approach. "All files" should be marked as "h1," and "Documents" should be marked as "h2." It's important to note that we should not have two "h1" headings on the page.

**Comment by JuliaKirschenheuter** [comment]
Thank you @michaelnissenbaum!

**Comment by JuliaKirschenheuter** [comment]
Dear @michaelnissenbaum,

i've checked this part and "Documents" is only a heading inside of the document which can be edited and is located inside of Text app. Whole content of the document is controlled by toolbar panel and there could be even several `<h1>` elements because user wanted to design a markdown document like this: 

![Screenshot from 2024-01-12 13-38-08](https://github.com/nextcloud/server/assets/6078378/1313e4d1-e180-40cc-b1a2-a153b6ac7047)

...or user can desing a document like: `<h4>` -> `<h1>` -> `<h5>` -> `<h2>` -> this is unfortunately out of our control:

![Screenshot from 2024-01-12 13-52-38](https://github.com/nextcloud/server/assets/6078378/d7065161-3702-408c-82d2-7bf8997de486)


Besides this a page itself has "all files" heading which comes from a left sidebar:

![Screenshot from 2024-01-12 13-45-08](https://github.com/nextcloud/server/assets/6078378/da9b2c9e-12b0-4169-9968-cac16c766da9)


I don't know how to solve this problem. Text app is only the one place where we have a such situation.

**Comment by michaelnissenbaum** [comment]
@JuliaKirschenheuter The text within the editor should not be a problem since it is the responsibility of the creator.

**Comment by JuliaKirschenheuter** [comment]
Thank you @michaelnissenbaum!

Can i then close this issue?

**Comment by michaelnissenbaum** [comment]
yes you can.

**Comment by JuliaKirschenheuter** [comment]
https://github.com/nextcloud/server/issues/42275#issuecomment-1893811697

**Comment by szaimen** [comment]
Chanted to close as not planned since we don't need to fix it
